src/windshaft/named-map.js

```diff
@@ -13,13 +14,18 @@ var NamedMap = MapBase.extend({
     //       "2": "/** torque visualization */\n\nMap { ... }"
     //     }
     //   }
-    //
-    json.styles = this._layersCollection.reduce(function (p, c, i) {
-      var style = c.get('cartocss');
+
+    var layers = this._layersCollection.filter(function (layer) {
+      return layer.get('type') !== 'GMapsBase';
```

Comment:
@nobuti TRUE! I forgot about the indexes... 😓 

But something like `_.reduce(this._layersCollection.getCartoDBLayers(), function (...) { ... })` should work, right?